Model/Button/GetCartButtonConfig.php

```diff
@@ -53,32 +56,76 @@ public function execute(
             $buttonConfig->setCartId($cartId);
         }
         $this->addGenericButtonConfig($buttonConfig, $refIdData, $quote, $customerSession, $initialAddress);
+        if ($triggerContext === 'regular-checkout') {
```

Comment:
Let's maybe introduce a constant here for maintainability (defined together with the other constants for trigger context):
```suggestion
        if ($triggerContext ===  ExpressCheckoutButton::TRIGGER_CONTEXT_REGULAR_CHECKOUT) {
```